app/assets/stylesheets/donor.css.scss

```diff
@@ -236,3 +236,12 @@
     font-weight: 500;
   }
 }
+
+.best_in_place:hover {
```

Comment:
I just had to take out the padding for when you hover so its a little uglier but it doesnt spaz now

![screen shot 2015-04-08 at 6 52 51 pm](https://cloud.githubusercontent.com/assets/1501088/7059137/7e079d5c-de20-11e4-93b1-81e77f3b1821.png)